standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestListPartitions.java

```diff
@@ -228,7 +233,7 @@ private void assertPartitionsHaveCorrectValues(List<Partition> partitions,
     }
   }
 
-  private void assertCorrectPartitionNames(List<String> names,
+  protected void assertCorrectPartitionNames(List<String> names,
```

Comment:
Why was this function type changed to protected?

Reply:
It's used in the child class `TestListPartitionsWithXIncludeParams`:
https://github.com/apache/hive/blob/84583f6fa1c779908852c49510c03495534f2e03/ql/src/test/org/apache/hadoop/hive/metastore/TestListPartitionsWithXIncludeParams.java#L145